test/mpi/main.cc

```diff
@@ -98,15 +117,25 @@ static ucc_coll_type_t coll_str_to_type(std::string coll)
 
 static ucc_memory_type_t mtype_str_to_type(std::string mtype)
 {
+    ucc_memory_type_t mem_type;
+
     if (mtype == "host") {
-        return UCC_MEMORY_TYPE_HOST;
+        mem_type = UCC_MEMORY_TYPE_HOST;
     } else if (mtype == "cuda") {
-        return UCC_MEMORY_TYPE_CUDA;
+        mem_type = UCC_MEMORY_TYPE_CUDA;
     } else {
         std::cerr << "incorrect memory type: " << mtype << std::endl;
         PrintHelp();
     }
-    abort();
+    if (UCC_MEMORY_TYPE_HOST != mem_type && UCC_OK != ucc_mc_available(mem_type)) {
+        std::cerr << "requested memory type '"
+                  << ucc_memory_type_names[mem_type]
+                  << "' is not support "
```

Comment:
not supportED, also there are extra " ' " single quotes.

Reply:
fixed